server/paymentProviders/stripe/payment-intent.ts

```diff
@@ -5,12 +5,18 @@ import type Stripe from 'stripe';
 import OrderStatuses from '../../constants/order_status';
 import logger from '../../lib/logger';
 import { getApplicationFee } from '../../lib/payments';
+import { reportMessageToSentry } from '../../lib/sentry';
 import stripe, { convertToStripeAmount } from '../../lib/stripe';
 import models from '../../models';
 
-import { APPLICATION_FEE_INCOMPATIBLE_CURRENCIES, refundTransaction, refundTransactionOnlyInDatabase } from './common';
+import {
+  APPLICATION_FEE_INCOMPATIBLE_CURRENCIES,
+  createChargeTransactions,
+  refundTransaction,
+  refundTransactionOnlyInDatabase,
+} from './common';
 
-const processOrder = async (order: typeof models.Order): Promise<void> => {
+async function processNewOrder(order: typeof models.Order) {
```

Comment:
Interesting, why hoisting?
Important to guarantee this returns undefined. Otherwise, it will break the expected behavior in:
https://github.com/opencollective/opencollective-api/blob/main/server/lib/payments.js#L577-L578

Reply:
What do you mean by hoisting?

Reply:
Declaring `processOrder` as a function and not as constant arrow function.

Reply:
whats the difference?

Reply:
LOL, that's what I'm questioning... 😂
I just wanted to know what was the rationale behind changing how we declare this function.
I'm fine with both.

Reply:
In this context, I think the difference for me its just a style choice. 😛  Thanks for pointing hoisting though, I didnt know that behavior in javascript!